docs/UserGuide.md

```diff
@@ -333,49 +404,49 @@ Format: `delProduct i/INDEX`
 * The index **must be a positive integer** 1, 2, 3, …​
 
 <div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
-This feature should only be used if there is a need to delete a product, which is unlikely in most conceivable scenario! Use with caution as this not only removes the product from MyInsuRec's product list, it also removes the product from any association with your clients.
+This feature should only be used if there is a need to delete a product, which is unlikely in most scenarios! Use with caution as this not only removes the product from MyInsuRec's product list, it also removes the product from any association with your clients.
```

Comment:
Could proabably bold the last sentence `removes the product from any association with your clients.`